src/main/java/com/radixdlt/crypto/KeyHandler.java

```diff
@@ -56,4 +56,11 @@
 	 * @throws CryptoException If the {@code privateKey} is invalid
 	 */
 	byte[] computePublicKey(byte[] privateKey) throws CryptoException;
+
+	default ECDSASignature sign(byte[] hash, byte[] privateKey) throws CryptoException {
+		boolean enforceLowS = true;
+		boolean useDeterministicSignatures = false;
+		return sign(hash, privateKey, enforceLowS, useDeterministicSignatures);

```

Comment:
Would be cleaner if these were just inlines, imo this actually makes it harder to read (though I appreciate what you're trying to do here)

Reply:
yeah using an IDE the usage of local variables just clutters the code. But not using an IDE I wanted to highlight what the boolean values meant, because Java is a crappy language without argument labels :D, but sure, I will remove them.

Reply:
Adding documentation as well.